service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <service verb="validate" noun="PostalAddress">
+            <implements service="mantle.shipment.CarrierServices.validate#ShippingPostalAddress"/>
+        <actions>
+            <!-- Searching FedexApiToken -->
+            <entity-find-one entity-name="ShippingGatewayOption" value-field="apiTokenOpt">
+                    <field-map field-name="shippingGatewayConfigId" from="shippingGatewayConfigId"/>
+                    <field-map field-name="optionEnumId" value="FedexApiToken"/>
+            </entity-find-one>
+            <if condition="!apiTokenOpt.optionValue">
+                <return type="warning" message=" ${shippingGatewayConfigId} has no FedexApiToken, not validating address"/>
+            </if>
+
+            <!-- Converting Address Data to JSON Request Map -->
+            <service-call name="mantle.fedex.FedexServices.get#AddressMap" in-map='[contactMechId:contactMechId, partyId:partyId]'
+                    out-map='addressMap'/>
```

Comment:
Please be consistent with the formatting, single quotes work but use double quotes to align with OOTB code patterns.